chart/templates/deployment.yaml

```diff
@@ -140,6 +140,10 @@ spec:
               name: "bootstrap-secret"
               key: "bootstrapPassword"
 {{- end }}
+{{- if (include "rancher.agent_tls_mode" .) }}
+        - name: CATTLE_AGENT_TLS_CHECK
```

Comment:
This should be `CATTLE_AGENT_TLS_MODE` and should be the same string value as was set in the chart.